build/common.sh

```diff
@@ -589,6 +589,11 @@ function kube::build::run_build_command_ex() {
     --env "GOGCFLAGS=${GOGCFLAGS:-}"
   )
 
+  if [ ! -z "${DOCKER_CGROUP_PARENT:-}" ]; then
```

Comment:
nit: we prefer the bash `[[ ]]` style over plain `[ ]`

Reply:
Done 👍